src/testing/account.ts

```diff
@@ -1,37 +1,35 @@
 import algosdk from 'algosdk'
-import { Config, getDispenserAccount } from '../'
-import { transferAlgos } from '../transfer/transfer-algos'
+import { AlgorandClient, Config } from '../'
 import { GetTestAccountParams } from '../types/testing'
 import Account = algosdk.Account
-import Algodv2 = algosdk.Algodv2
-import Kmd = algosdk.Kmd
 
 /**
  * Creates an ephemeral Algorand account for the purposes of testing.
  * Returns a newly created random test account that is funded from the dispenser
  * DO NOT USE THIS TO CREATE A MAINNET ACCOUNT!
  * Note: By default this will log the mnemonic of the account.
  * @param param0 The config for the test account to generate
- * @param algod An algod client
- * @param kmd A KMD client, if not specified then a default KMD client will be loaded from environment variables
+ * @param algorand An AlgorandClient client
  * @returns The account, with private key loaded
  */
 export async function getTestAccount(
   { suppressLog, initialFunds, accountGetter }: GetTestAccountParams,
-  algod: Algodv2,
-  kmd?: Kmd,
```

Comment:
Technically this is a breaking change, but I'd be very surprised if anyone used this method directly - you have to go out of your way to import it (it's not in the default `* as algokit` export. The way it's exposed via `algorandFixture` isn't breaking.